Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -197,3 +199,32 @@ The erasure coded pool must be set as the `dataPool` parameter in
 The erasure coded pool must be set as the `dataBlockPool` parameter in
```

Comment:
Please remove this whole section on the Flex Driver (lines 199-201). It's very stale information.